docs/modules/traverse.md

```diff
@@ -0,0 +1,47 @@
+# traverse
+
+Traverse and transform objects by visiting every node on a recursive walk.
```

Comment:
This should instead be brief wording on why we need an alternative (e.g. "lighter alternatives exist")